source/features/clean-conversation-headers.css

```diff
@@ -61,3 +64,7 @@
 .rgh-clean-conversation-headers .commit-ref-dropdown {
 	margin-top: -2px !important;
 }
+
+.d-flex:has(.rgh-clean-conversation-headers) {
```

Comment:
This is very broad, it can target any flex element wrapping the whole page. Maybe you want `.d-flex:has(> .rgh-clean-conversation-headers)` and/or even better prepend some other class like `.whatever .d-flex:has(.rgh-clean-conversation-headers)`

But actually it's likely easier to just add `.flex-items-center` to the element, like:

```js
byline.closest('.d-flex')!.add('flex-items-center')
```

:has() can be fatal to performance so it needs to be used cautiously:

- https://github.com/refined-github/refined-github/issues/7331#issuecomment-2120464245